x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/action/FindFileStructureAction.java

```diff
@@ -148,6 +141,27 @@ public boolean equals(Object other) {
         public Request() {
         }
 
+        public Request(StreamInput in) throws IOException {
+            super(in);
+            linesToSample = in.readOptionalVInt();
+            if (in.getVersion().onOrAfter(Version.CURRENT)) {
```

Comment:
unrelated to this PR: this looks like a bug, we should never be comparing to Version.CURRENT since that will change...

Reply:
I opened https://github.com/elastic/elasticsearch/pull/44316 for this